Demo/Application/Features/SEPA Direct Debit/BraintreeDemoSEPADirectDebitViewController.swift

```diff
@@ -81,5 +81,65 @@ class BraintreeDemoSEPADirectDebitViewController: BraintreeDemoBaseViewControlle
     private func generateRandomCustomerID() -> String {
         String(UUID().uuidString.replacingOccurrences(of: "-", with: "").prefix(20))
     }
+
+    // MARK: - Helper functions for IBAN generation
+
+    private func generateRandomIBAN() -> String {
```

Comment:
Following up on our idea to have this live as public SDK helper func for merchants / something we can add in our docs to improve the testing story?